contentcuration/contentcuration/api.py

```diff
@@ -44,4 +56,32 @@ def get_child_names(node):
     names = []
     for n in node.get_children():
         names.append({"title": n.title, "id" : n.id})
-    return names
\ No newline at end of file
+    return names
+
+def delete_children(node):
+    for n in node.get_children():
+        delete_children(n)
+    node.delete()
+
+def batch_add_tags(request):
```

Comment:
I think we are doing this in the CustomListSerializer update().
This api method now does not get used by anyone.